wire-ios-utilities/Resources/WireUtilities-Info.plist

```diff
@@ -24,8 +24,6 @@
 	<string>Copyright © 2015 Wire. All rights reserved.</string>
```

Comment:
missing ref of submodule [d48bf72](https://github.com/wireapp/wire-ios-build-assets/commit/d48bf725061aabfd2f8d180240d10fdefd81d7f7)

Reply:
It was updating in the PR I was expecting to merge first, but I updated it here too